src/libs/dal/HSBContext.cs

```diff
@@ -103,26 +103,48 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
         modelBuilder.HasDbFunction(typeof(HSBContext)
             .GetMethod(
                 nameof(FindServerHistoryItemsByMonth),
-                new[] { typeof(DateTime), typeof(DateTime?), typeof(int?), typeof(int?), typeof(int?), typeof(string) })!
-            ).HasName("FindServerHistoryItemsByMonth");
-
+                [typeof(DateTime), typeof(DateTime?), typeof(int?), typeof(int?), typeof(int?), typeof(string)])!
+            ).HasName(nameof(FindServerHistoryItemsByMonth));
         modelBuilder.HasDbFunction(typeof(HSBContext)
             .GetMethod(
                 nameof(FindServerHistoryItemsByMonthForUser),
-                new[] { typeof(int), typeof(DateTime), typeof(DateTime?), typeof(int?), typeof(int?), typeof(int?), typeof(string) })!
-            ).HasName("FindServerHistoryItemsByMonthForUser");
-
+                [typeof(int), typeof(DateTime), typeof(DateTime?), typeof(int?), typeof(int?), typeof(int?), typeof(string)])!
+            ).HasName(nameof(FindServerHistoryItemsByMonthForUser));
         modelBuilder.HasDbFunction(typeof(HSBContext)
             .GetMethod(
                 nameof(FindFileSystemHistoryItemsByMonth),
-                new[] { typeof(DateTime), typeof(DateTime?), typeof(int?), typeof(int?), typeof(int?), typeof(string) })!)
-            .HasName("FindFileSystemHistoryItemsByMonth");
-
+                [typeof(DateTime), typeof(DateTime?), typeof(int?), typeof(int?), typeof(int?), typeof(string)])!
+            ).HasName(nameof(FindFileSystemHistoryItemsByMonth));
         modelBuilder.HasDbFunction(typeof(HSBContext)
             .GetMethod(
                 nameof(FindFileSystemHistoryItemsByMonthForUser),
-                new[] { typeof(int), typeof(DateTime), typeof(DateTime?), typeof(int?), typeof(int?), typeof(int?), typeof(string) })!)
-            .HasName("FindFileSystemHistoryItemsByMonthForUser");
+                [typeof(int), typeof(DateTime), typeof(DateTime?), typeof(int?), typeof(int?), typeof(int?), typeof(string)])!
+            ).HasName(nameof(FindFileSystemHistoryItemsByMonthForUser));
+
+        // modelBuilder.Entity<ServerHistoryItemsByMonth>().ToTable((string?)null);
```

Comment:
Various attempts to get Entity Framework to allow mapping the stored procedures to other objects.  I was unsuccessful, but hope to figure this out still.